src/Altinn.Apps/AppTemplates/AspNet/App.IntegrationTests/Data/apps/tdd/autodelete-true/appsettings.Development.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
is the test dependent on this file or could it be removed??

Reply:
Removed